adsyncdump.py

```diff
@@ -0,0 +1,15 @@
+from havoc import Demon, RegisterCommand
+
+def adsyncdump(demonID, *param):
+    TaskID : str    = None
+    demon  : Demon  = None
+
```

Comment:
These definitions don't make sense as the type hint is wrong and you immediately overwrite them later, i'd just remove these two lines all together